saleor/plugins/webhook/tasks.py

```diff
@@ -272,6 +273,8 @@ def trigger_all_webhooks_sync(
                 request_context = initialize_request(
                     requestor, event_type in WebhookEventSyncType.ALL
                 )
+                request_context.is_mutation = is_mutation
```

Comment:
Why we don't set this inside `initialize_request`? It looks like this is a perfect function to set some flags on the request.

Reply:
We shouldn't set this flag for all webhooks. It's required only for synchronous webhooks. 
Maybe we should add the arg to `initialize_request` e.g. `use_replicas`, and set it by default to `True`.

Reply:
Yeah, but `is_mutation` is boolean, so you can by default set it to `False` and change it only for sync webhooks to `True`.